packages/react-components/src/unstable/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@fluentui/react-components",
+  "description": "Separate entrypoint for unstable Fluent UI components",
```

Comment:
```suggestion
```

We don't need this, https://cdn.jsdelivr.net/npm/rtl-css-js@1.14.2/core/package.json

Reply:
I'll leave the description there, since it's essentially documentation